src/FirstSteps/RPG/Items/Item.cs

```diff
@@ -13,6 +13,10 @@ public Item(string name, int price, int weight)
             Weight = weight;
         }
 
+        public Item()
```

Comment:
Do you need this? xD
Try to review your PR at first because you can catch things like that.